js/src/panes/pane.fullscreen.js

```diff
@@ -78,9 +78,7 @@ Wu.Fullscreen = Wu.Evented.extend({
 
 		var closeCallback = this.options.closeCallback;
 		closeCallback && closeCallback();
-		Wu.Mixin.Events.fire('closeFullscreen', {detail : {
-			projectId: options.projectId || null
```

Comment:
need to add projectId here if it was there before, some listeners may make use of it.